framework/src/binding/state/trie_db.rs

```diff
@@ -26,8 +34,21 @@ impl RocksTrieDB {
         Ok(RocksTrieDB {
             light,
             db: Arc::new(db),
+            cache: RwLock::new(HashMap::new()),
```

Comment:
If the HashMap triggers resort, performance would suffer greatly because it would need to copy all the data once.

So, It is best to capacity for enough size in advance.